### PR TITLE
[FEAT] join event participants component

### PR DESCRIPTION
**Pull request by Chiranjibichapagain**

### CODE REVIEW

File: client/src/hooks/useEventParticipants.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const useEventParticipants = (eventId: EventId) => {
 
   useEffect(() => {
     fetchParticipants()
-  }, [])
+  }, [participants])
```
Comment (MikeVedsted) [inline client/src/hooks/useEventParticipants.tsx:11]: you are suffering from the same issue as Nanna was - your dependencies are causing the hook to fire constantly asd as a result bombarding the DB with requests.

**Review by MikeVedsted — APPROVED** [review]

File: client/src/components/EventParticipantsAndRequests/index.tsx [hunk 2]
```diff
@@ -35,21 +35,33 @@ const EventParticipantsAndRequests = ({
...
+            participants.map((participant: Participant) => (
+              <EventParticipant
+                key={participant.user_id}
+                participant={participant}
+              />
+            ))
+          ) : (
+            <h2 className='participants__not-found'>
+              Sorry! No participants yet.
+            </h2>
```
[the remark this answers is not in the document]
Comment (Chiranjibichapagain) [reply client/src/components/EventParticipantsAndRequests/index.tsx:48]: :+1:

File: client/src/components/EventParticipantsAndRequests/index.tsx [hunk 3]
```diff
@@ -35,21 +35,33 @@ const EventParticipantsAndRequests = ({
...
+              <EventJoinRequest
+                key={participant.user_id}
+                requester={participant}
+              />
+            ))
+          ) : (
+            <h2 className='participants__not-found'>
+              Sorry! No join requests yet.
+            </h2>
+          )}
```
[the remark this answers is not in the document]
Comment (Chiranjibichapagain) [reply client/src/components/EventParticipantsAndRequests/index.tsx:64]: :+1:

File: client/src/components/EventParticipantsAndRequests/EventParticipantsAndRequests.scss [hunk 4]
```diff
@@ -19,6 +17,13 @@
     margin-bottom: 20px;
   }
 
+  &__not-found {
+    font-family: $nunito;
+    font-size: $text;
+    color: $secondary;
+    text-align: center;
+  }
```
[the remark this answers is not in the document]
Comment (Chiranjibichapagain) [reply client/src/components/EventParticipantsAndRequests/EventParticipantsAndRequests.scss:25]: :+1:

File: client/src/hooks/useEventParticipants.tsx (hunk 1 above)
Comment (Chiranjibichapagain) [reply client/src/hooks/useEventParticipants.tsx:11]: I Will remove but it won't update the list immediately..

**Review by Rostislav013 — APPROVED** [review]

**Pull request merged**